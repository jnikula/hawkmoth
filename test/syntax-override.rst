
C++ class inside a '.h'.

