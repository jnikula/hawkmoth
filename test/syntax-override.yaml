domain: cpp
directive-arguments:
  - wrong-syntax.h
directive-options:
  clang:
    - -xc++
example-title: C++ syntax override for .h header
example-priority: 1
