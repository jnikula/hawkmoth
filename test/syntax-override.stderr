WARNING: wrong-syntax.h:2: documentation comment attached to unexpected cursor CursorKind.CLASS_DECL foo
